test/e2e/fixture.go

```diff
@@ -254,6 +254,116 @@ func exportLogs(tb testing.TB, data *TestData, logsSubDir string, writeNodeLogs
 	}
 }
 
+func exportMainTestLogs(data *TestData, logsSubDir string, writeNodeLogs bool) {
+	const timeFormat = "Jan02-15-04-05"
+	timeStamp := time.Now().Format(timeFormat)
+	logsDir := filepath.Join(data.logsDirForTestCase, fmt.Sprintf("%s.%s", logsSubDir, timeStamp))
+	err := createDirectory(logsDir)
+	if err != nil {
+		log.Printf("Error when creating logs directory '%s': %v", logsDir, err)
+		return
+	}
+	log.Printf("Exporting test logs to '%s'", logsDir)
+
+	// getPodWriter creates the file with name nodeName-podName-suffix. It returns nil if the
+	// file cannot be created. File must be closed by the caller.
+	getPodWriter := func(nodeName, podName, suffix string) *os.File {
```

Comment:
Oh actually all lines are duplicated from function `exportLogs()`, probably better to add an option for function `exportLogs()` rather than creating a new function?

Reply:
Yes, they are duplicated. The tricky thing is, `exportLogs` has an argument `tb testing.TB`, and TestMain doesn't have this interface. So for all the occurrence of `tb` in the function, we'll need to add a if statement. e.g. `tb.Logf` -> `log.Printf`. Do you have any suggestions?

Reply:
Thanks, looks much better now.